src/base/users/View.svelte

```diff
@@ -161,96 +161,96 @@
         </div>
       </div>
     </header>
-      <div class="fields">
-        <!-- Address -->
-        <div class="label">Address</div>
-        <div class="desktop"><Address {config} address={profile.address} /></div>
-        <div class="mobile"><Address compact {config} address={profile.address} /></div>
+    <div class="fields">
```

Comment:
Here we had wad seems like a wrong indentation, feels great to find them myself :sweat_smile: